docs/Reference/Server.md

```diff
@@ -1048,6 +1049,21 @@ fastify.routing(req, res)
 Method to add routes to the server, it also has shorthand functions, check
 [here](./Routes.md).
 
+#### hasRoute
+<a id="hasRoute"></a>
+
+Method to check if a route is already registered to the internal router:
```

Comment:
I don't know what happened to my other comment, but these docs do not describe the required input object. What properties are possible? Which ones are required? The method signature should be fully documented.

Reply:
Do you have an example in the docs, so that i can do it with the same style?

Reply:
@jsumners 
better?